### PR TITLE
Add lease support to stressbench tests

### PR DESCRIPTION
**Pull request by jiacheliu3**
These two bench tests did not respect register lease relevant configurations.

### CODE REVIEW

**Comment by jiacheliu3** [comment]
@Xenorith This is a recently found bug. If possible let's add it to 2.7. It's also alright if it's missed the bus.

**Review by dbw9580 — APPROVED** [review]
LGTM!

**Comment by jiacheliu3** [comment]
alluxio-bot, check this please

File: stress/shell/src/main/java/alluxio/stress/cli/StreamRegisterWorkerBench.java [hunk 1]
```diff
@@ -98,6 +100,11 @@ public void prepare() throws Exception {
                     .newBuilder(ClientContext.create(mConf))
                     .build());
     mBlockMap = blockMap;
+    int blockCount = 0;
+    for (Map.Entry<BlockStoreLocation, List<Long>> entry : blockMap.entrySet()) {
+      blockCount += entry.getValue().size();
+    }
+    mBlockCount = blockCount;
```
[the remark this answers is not in the document]
Comment (jiacheliu3) [reply stress/shell/src/main/java/alluxio/stress/cli/StreamRegisterWorkerBench.java]: done

**Review by dbw9580 — APPROVED** [review]

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Comment by Xenorith** [comment]
doh not merged yet. gotta wait

**Review by ggezer — APPROVED** [review]

**Comment by jiacheliu3** [comment]
alluxio-bot, merge this please

**Comment by jiacheliu3** [comment]
@Xenorith all yours :)

**Comment by Xenorith** [comment]
alluxio-bot, cherry-pick this to branch-2.7 please

**Pull request merged**